package.json

```diff
@@ -44,8 +44,8 @@
     "dev": "webpack-dev-server --inline --progress --host 0.0.0.0 --config build/webpack.dev.conf.js",
     "dev:server": "nodemon bin/www",
     "start": "node bin/www",
-    "lint": "eslint --ext .js,.vue .",
-    "lint-fix": "eslint --fix --ext .js,.vue .",
+    "lint": "eslint --ext .js,.vue . && markdownlint '**/*.md'",
```

Comment:
Usually I prefer to split scripts like `lint:markdown` and `lint:eslint` and then have `lint: "npm run lint:markdown && npm run lint:eslint"`